src/main/java/backend/resource/TurboIssue.java

```diff
@@ -391,6 +393,22 @@ public TurboIssue combineWithPullRequest(PullRequest pullRequest) {
         return getMatchedIssues(issues, query).stream().findFirst();
     }
 
+    /**
+     * @param queries
+     * @return number of matches to a list of query
+     */
+    private int getNumberOfMatchingQuery(List<String> queries) {
```

Comment:
Maybe this could be named `getMatchingQueryCount`? It's the number of queries which would match this issue. That can be in the comment as well possibly.